packages/rocket-file-uploads-types/src/constants.ts

```diff
@@ -1,2 +1,6 @@
 export const storageName = (applicationName: string, environmentName: string): string =>
   `${applicationName}rf${environmentName}`.replace(/([-_])/gi, '').substr(0, 24)
+
+export const LOCAL_PORT_ENV = 'ROCKET_FILES_LOCAL_PORT'
+
+export const localPort: string = process.env[LOCAL_PORT_ENV] || '3000'
```

Comment:
I think this should be a function, if not, since this file is first required, the value of this constant will be calculated and, as the env var ROCKET_FILES_LOCAL_PORT will be empty in that moment (as it is set in the function mountStack), the constant will always be 3000 from that moment on.

Using a function here would resolve that problem:
```suggestion
export const localPort = (): string => process.env[LOCAL_PORT_ENV] || '3000'
```